controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/Role.java

```diff
@@ -43,31 +43,16 @@ public static TenantRole athenzTenantAdmin(TenantName tenant) {
         return new TenantRole(RoleDefinition.athenzTenantAdmin, tenant);
     }
 
-    /** Returns a {@link RoleDefinition#tenantPipeline} for the current system and given tenant and application. */
-    public static ApplicationRole tenantPipeline(TenantName tenant, ApplicationName application) {
-        return new ApplicationRole(RoleDefinition.tenantPipeline, tenant, application);
+    /** Returns a {@link RoleDefinition#integrationTest} for the current system and given tenant and application. */
+    public static ApplicationRole integrationTest(TenantName tenant, ApplicationName application) {
+        return new ApplicationRole(RoleDefinition.integrationTest, tenant, application);
```

Comment:
On second thought, the integration test is a `hostedOperator`, so this isn't really needed.